DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/convolution2d/src/convolution_kernel.hpp

```diff
@@ -294,6 +295,7 @@ struct Grey2RGB {
   void operator()() const {
     // this loop is not necessary in hardware since the `start` bit will be
     // pulled high, but it makes the testbench easier.
+    [[intel::initiation_interval(1)]]  //
```

Comment:
```suggestion
    [[intel::initiation_interval(1)]]
```

Reply:
The // is there to prevent clang-format from messing with the attributes. without it, the formatter moves the attribute and while loop to the same line.

Reply:
In that case, I've updated my suggestion to match the other samples.